modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/EditAPIEndpoint.tsx

```diff
@@ -50,11 +51,11 @@ export default function EditAPIEndpoint({
 }: EditAPIEndpointProps) {
 	const {
 		fetchedData,
+		isDataUnsaved,
```

Comment:
@cgmonte I'm not sure if "unsaved" exists. Maybe `isDataNotSaved`?

Reply:
yes, it exists https://www.merriam-webster.com/dictionary/unsaved